csp_billing_adapter_local/plugin.py

```diff
@@ -111,3 +114,57 @@ def save_csp_config(
 ):
     """Save specified content as local storage csp_config contents."""
     update_csp_config(config, csp_config, replace=True)
+
+
+@csp_billing_adapter.hookimpl(trylast=True)
+def get_usage_data(config: Config):
+    """
+    Retrieves the current usage report from API
+
+    :param config: The application configuration dictionary
+    :return: Return a dict with the current usage report
+    """
+    resource = _make_request(config.get('api'))
+    resource_key = list(resource.keys())
+    if resource_key and resource_key[0] in config:
+        resource_key = resource_key[0]
```

Comment:
First `resource_key` is a `list` and then  here we make it become a string that is then used to access information inside the structure we received form the application. Everything starts with `resource` making the code rather confusing and prone to human error in reading and understanding. `resource` and `resource_key` are very similar and the reader can easily make a mistake.